src/StagingBanner.jsx

```diff
@@ -3,9 +3,15 @@ import { Portal } from 'react-portal';
 import './less/stagingBanner.less';
 import config from '@plone/volto/registry';
 import cx from 'classnames';
+import { runtimeConfig } from '@plone/volto/runtime_config';
+import { Icon } from '@plone/volto/components';
 
 const StagingBanner = () => {
   const stagingBannerConfig = config.settings.stagingBanner;
+  const bannerHeader =
+    runtimeConfig[config.settings.stagingBanner.envBannerHeader];
```

Comment:
You have stagingBannerConfig saved above, use it to reference the keys to the runtime config:
runtimeConfig[stagingBannerConfig.envBannerHeader]

Reply:
Done!